README.md

```diff
@@ -14,36 +14,37 @@ For Amazon S3:
 
 ```
 docker run --rm \
-  -v $(pwd):/backup
-  -e NAME=backup-name \
+  -v /path/to/backups/locally:/backup \
+  -e NAME=<compressed-file-name> \
   -e AWS_ACCESS_KEY=xxx \
   -e AWS_SECRET_KEY=xxx \
-  -e S3_PATH=bucket/directory \
-  caseycs/dir-backup-s3
+  -e S3_PATH=<bucket-name>/<directory-for-backups> \
+  freefly19/dir-backup-s3
 ```
 
 For DigitalOcean Spaces:
 
 ```
 docker run --rm \
-  -v $(pwd):/backup
-  -e NAME=backup-name \
+  -v /path/to/backups/locally:/backup \
+  -e NAME=<compressed-file-name> \
   -e AWS_ACCESS_KEY=xxx \
   -e AWS_SECRET_KEY=xxx \
-  -e S3_PATH=bucket/directory \
-  -e S3CMD_OPTIONS="--host=ams3.digitaloceanspaces.com --host-bucket=%(bucket)s.ams3.digitaloceanspaces.com" \
-  caseycs/dir-backup-s3
+  -e S3_PATH=<bucket-name>/<directory-for-backups> \
+  -e S3CMD_OPTIONS="--host=ams3.digitaloceanspaces.com --host-bucket=$(bucket).ams3.digitaloceanspaces.com" \
+  freefly19/dir-backup-s3
```

Comment:
Please keep original image name so i'll be able to merge the PR.
You can always refer to your image name inside branch/master of your own fork.